config-templates/redhat/local-volume/4.5/custom-parameters.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "description": "Node label key",
+    "displayname": "Node Label Key",
+    "name": "label-key",
+    "required": true,
```

Comment:
Add all `true`/`false` in the double quotes

Reply:
replaced all `"true"`/`"false"` with `"true"`/`"false"`